emails/tests/mgmt_send_welcome_emails.py

```diff
@@ -0,0 +1,100 @@
+import pytest
```

Comment:
This file should be named `mgmt_send_welcome_emails_tests`, so that `pytest` will run it with the other tests.